packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -1390,52 +1430,36 @@ export class IntervalCollection<TInterval extends ISerializableInterval>
 
 	/**
 	 * {@inheritdoc IIntervalCollection.changeProperties}
+	 * @deprecated - call change with the id and an object containing the new props values
 	 */
 	public changeProperties(id: string, props: PropertySet) {
```

Comment:
Putting this here as not a better place for it: we will want some API updates to `IIntervalCollectionEvent`.

There's not a super great way to stage semantic changes to that interface, so it might make sense to provide an alternative API to subscribe to events on which provides semantics more aligned with the fact that `change` and `changeProperties` are unified. Since that'll be a new API, we can do that in a separate PR.

Depending on usage, we could also just not provide a super simple migration path which allows update at leisure and make the change in a breaking version. But that experience is much worse for customers if they have extensive usage.